core/io/image.cpp

```diff
@@ -1828,6 +1828,12 @@ void Image::_generate_mipmap_from_format(Image::Format p_format, const uint8_t *
 				_generate_po2_mipmap<uint8_t, 4, false, Image::average_4_uint8, Image::renormalize_uint8>(p_src, p_dst, p_width, p_height);
 			}
 		} break;
+		case Image::FORMAT_RGBA4444: {
+			_generate_po2_mipmap<uint16_t, 1, false, Image::average_4_rgba4444, Image::renormalize_rgba4444>(src_u16, dst_u16, p_width, p_height);
```

Comment:
```suggestion
			_generate_po2_mipmap<uint16_t, 1, false, Image::average_4_rgba4444, nullptr>(src_u16, dst_u16, p_width, p_height);
```

If it's not used and never called, what's the purpose of having a function? It's just making it less readable. Same for the other one and existing `renormalize_rgbe9995`.